.travis.yml

```diff
@@ -7,6 +7,7 @@ php:
   - 7.0
   - 7.1
   - 7.2
+  - 7.3
```

Comment:
The `php-7.4` version is stable now.

I think it should add `php-7.4` version test on Travis CI build.

Reply:
Sure. Time flies!